.github/workflows/release.yml

```diff
@@ -83,9 +83,13 @@ jobs:
             const url = require('url');
             const https = require('https');
             checksums = {}
+            var noReleaseMatched = true;
+            var noAssetsFound = true;
```

Comment:
So this comment *can* be ignored. But my own personal preference is to have variables be worded so that there isn't double negation --> i.e noAssetsFound = false is a little confusing to me. Whereas assetsFound = true / false is simple to understand.

Reply:
Yeah! Definitely, that will be simple to understand. I will update it.

Reply:
Adopted.